streams-runtimes/streams-runtime-local/src/main/java/org/apache/streams/local/builders/LocalStreamBuilder.java

```diff
@@ -124,6 +127,16 @@ public void run() {
         this.futures = new HashMap<>();
     }
 
+    public void setQueueCapacity(Map<String, Object> streamConfig) {
+        try {
+            if (streamConfig.containsKey("maxQueueSize") && Integer.valueOf((Integer) streamConfig.get("maxQueueSize")) > 0) {
```

Comment:
Correct, I suppose I was nervous that the default values might not always be respected.